src/OpenDebugAD7/AD7DebugSession.cs

```diff
@@ -705,22 +704,8 @@ protected override void HandleLaunchRequestAsync(IRequestResponder<LaunchArgumen
                 // Don't convert the workingDirectory string if we are a pipeTransport connection. We are assuming that the user has the correct directory separaters for their target OS
                 string workingDirectoryString = pipeTransport != null ? workingDirectory : m_pathConverter.ConvertClientPathToDebugger(workingDirectory);
 
-                bool debugServerUsed = false;
-                bool isOpenOCD = false;
-                bool stopAtEntrypoint;
-                bool visualizerFileUsed;
-                string launchOptions = MILaunchOptions.CreateLaunchOptions(
-                    program: program,
-                    workingDirectory: workingDirectoryString,
-                    args: JsonConvert.SerializeObject(responder.Arguments.ConfigurationProperties),
-                    isPipeLaunch: responder.Arguments.ConfigurationProperties.ContainsKey("pipeTransport"),
-                    stopAtEntry: out stopAtEntrypoint,
-                    isCoreDump: out m_isCoreDump,
-                    debugServerUsed: out debugServerUsed,
-                    isOpenOCD: out isOpenOCD,
-                    visualizerFileUsed: out visualizerFileUsed);
-
-                m_sessionConfig.StopAtEntrypoint = stopAtEntrypoint;
+                m_isCoreDump = responder.Arguments.ConfigurationProperties.ContainsKey("coreDumpPath");
```

Comment:
>m_isCoreDump [](start = 16, length = 12)

I bet we could get rid of this variable -- all of the special casing it is used for looks useless.